spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+require 'coveralls'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.